doc/quick-install.md

```diff
@@ -1,92 +0,0 @@
-Installing BU from sources and from Ubuntu repos
```

Comment:
why remove the quick install?  Its often the most used doc :-)...

Reply:
The info was not deleted, but moved to /INSTALL.md

I can move it back to doc/quick-install.md if you prefer.

Reply:
@gandrewstone since basically what was the content of `doc/quick-install.md` has been moved on top of the `INSTALL.md` file I think this change increased the usability of the repo.